README.md

```diff
@@ -71,7 +71,7 @@ To run the project, execute the following steps:
 
 ### Deployment
 
-You can deploy this project using: google chrome.
+You can deploy this project using: https://leeoasis.github.io/My-Portfolio/
```

Comment:
[OPTIONAL] I think this statement can be worked on😉 The statement doesn't look correct because you are referencing to your live link and not a way to deploy your project. I suggest you can simply use `You can view this project live using:  https://leeoasis.github.io/My-Portfolio/`